include/tvm/ffi/function.h

```diff
@@ -878,6 +880,56 @@ inline int32_t TypeKeyToIndex(std::string_view type_key) {
         reinterpret_cast<::tvm::ffi::Any*>(result));                                   \
     TVM_FFI_SAFE_CALL_END();                                                           \
   }                                                                                    \
+  TVM_FFI_DLL_EXPORT const char* __tvm_ffi__metadata_##ExportName() {                  \
```

Comment:
this function needs to sit behind a flag TVM_FFI_DLL_EXPORT_TYPED_FUNC_META_DATA, so it can be toggled on off, mainly for concerns of binary size etc.